db/schema.rb

```diff
@@ -27,10 +36,21 @@
   end
 
   create_table "purchases", force: :cascade do |t|
-    t.integer  "user_id",    null: false
-    t.integer  "product_id", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.integer  "user_id",             null: false
+    t.integer  "product_id",          null: false
+    t.datetime "created_at",          null: false
+    t.datetime "updated_at",          null: false
+    t.integer  "purchasing_quantity"
```

Comment:
@danielpaul This is the part I did in Payment Integration I don't know why it's here... It's the column that record the quantity and price when purchase is happening.